eslint-example/main.js

```diff
@@ -0,0 +1,12 @@
+ // not supported in Firefox
+ // import { getUsefulContents } from "file";
```

Comment:
Interesting. In Firefox I see an error that suggests this *might* work. I don't think it will work yet though. I also tried it in Nightly, same error.